recipes/lapack/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# bug 145
+if test `uname` = "Linux"
+then
+  sed -e 's|/CMAKE/|/cmake/|' -i CBLAS/CMakeLists.txt
+fi
+
+# lapack_testing.py does not support py3k, correction sent
+cp ${RECIPE_DIR}/lapack_testing.py .
```

Comment:
Are we just copying this because the copy included here supports Python 2/3? I'm asking because I know this file already exists in a standard copy of LAPACK. Maybe we should just apply the patch that you submitted. No real strong opinion. Just a thought.

Also, do you have a link for your patch submission?